<launch>
    <node pkg="motiodom" exec="motiodom_node" name="motiodom_node">
        <remap from="/imu" to="/imu"/>
        
        <param name="enable_magnet" value="false"/>
        <param name="enable_position" value="false"/>
        <param name="frame_id" value="odom"/>
        <param name="child_frame_id" value="imu"/>
        <param name="delta_time" value="10"/>
    </node>
</launch>